core/pom.xml

```diff
@@ -51,15 +51,13 @@
       </dependency>
 
       <dependency>
-         <groupId>org.eclipse.microprofile.metrics</groupId>
-         <artifactId>microprofile-metrics-api</artifactId>
-         <optional>true</optional>
+         <groupId>io.micrometer</groupId>
+         <artifactId>micrometer-core</artifactId>
```

Comment:
I would make all of these dependencies optional. Micrometer should not be a requirement

Reply:
Right! I'll put it back.

Reply:
Done!